pkgs/by-name/sn/snort2/package.nix

```diff
@@ -1,70 +0,0 @@
-{
-  lib,
-  stdenv,
-  pkg-config,
-  luajit,
-  openssl,
-  fetchurl,
-  libpcap,
-  pcre,
-  libdnet,
-  daq,
-  zlib,
-  flex,
-  bison,
-  makeWrapper,
-  libtirpc,
-}:
-
-stdenv.mkDerivation (finalAttrs: {
-  version = "2.9.20";
```

Comment:
Can you add this to pkgs/top-level/aliases.nix as a removed package? Same as for the other one?